engine.py

```diff
@@ -85,7 +85,16 @@ def has_received_ui_creation_requests(self):
         via the show_dialog methods
         """
         return self._ui_created
-                
+
+    @property
+    def host_info(self):
+        """
+        :returns: A (name, release) tuple with informations about the
+                  application hosting this engine.
+        """
+        version = self.shotgun.info()["version"]
```

Comment:
Any concern here about weird SG dev branches? Just curious.

Reply:
Not sure... I changed it to `version = self.shotgun.info().get("version") or ["unknown"]`, would that cover the case you have in mind?